chut/__init__.py

```diff
@@ -233,6 +233,8 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
         if 'sh' in kwargs:
             kwargs['shell'] = kwargs.pop('sh')
+        if kwargs.get('shell') and len(args) > 1:
+            raise TypeError("One argument at most is allowed when shell=True")
```

Comment:
I don't really see the problem. Can be usefull to "escape" filenames with spaces

```
ls('-l "', weird_filename, '"')
```